netci.groovy

```diff
@@ -28,6 +28,9 @@ def static getBuildJobName(def configuration, def os) {
             if (os == 'Windows_NT') {
                 buildCommand = ".\\build.cmd -Configuration ${lowerConfiguration} Default"
             }
+            else if (os == 'Windows_2016') {
+                buildCommand = ".\\build.cmd -Configuration ${lowerConfiguration} -RunInstallerTestsInDocker Default"
+            }
```

Comment:
My plan was to turn on Windows 2016 and keep running Windows NT at the same time. Then when Windows 2016 is stable, remove Windows NT.

Does that make sense?